community/openebs/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+version: 0.7.0
+name: openebs
+appVersion: 0.7.0
+tillerVersion: '>=2.7.2'
+description: OpenEBS is an open source, cloud native storage for Kubernetes.
+icon: https://raw.githubusercontent.com/openebs/chitrakala/master/OpenEBS%20logo/openebs%20logos-03.png
+home: http://www.openebs.io/
+keywords:
+  - cloud-native-storage
+  - block-storage
+  - ICP
+  - IKS
+  - amd64
+  - OpenSource
+  - DevOps
+  - iSCSI
+  - storage

```

Comment:
My apologies, I forgot to mention this in my earlier comments, but this is another small change that will improve the experience on ICP 3.1. I would change `storage` to `Storage`. The ICP 3.1 user interface introduces categories, based on keywords, and it recognizes `Storage` as a category but not `storage`. Details on current keywords/categories are listed here: https://github.com/IBM/charts/blob/master/GUIDELINES.md#chart-keywords-1